src/components/FiveDayWeather.jsx

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+import WeatherCard from "./WeatherCard";
+import Grid from "@material-ui/core/Grid";
+
+const FiveDayWeather = ({ weather, unit }) => {
+  const hasWeather = Object.keys(weather).length > 0;
+
+  return (
+    <Grid container justify="center">
+      {hasWeather &&
+        weather.list.slice(0, 5).map((list, i) => (
+          <div key={i}>
+            <WeatherCard weather={list} unit={unit} />
+          </div>
+        ))}
+    </Grid>
+  );
+};
```

Comment:
Ideally, i would use the Daily Forecast endpoint setting the `cnt` param to `5` to retrieve 5 days of forecast but you have to pay for that endpoint: https://openweathermap.org/forecast16

As a shortcut, i'm hitting the 5 day weather forecast endpoint and slicing off the first 5. If i had more time i would parse through the data sent from the 5 day weather forecast and only display one data point for each day: https://openweathermap.org/forecast5